subcommands/ptar/ptar.1

```diff
@@ -0,0 +1,76 @@
+.Dd June 26, 2025
+.Dt PLAKAR-PTAR 1
+.Os
+.Sh NAME
+.Nm plakar ptar
+.Nd generate a self-contained Kloset archive (.ptar)
+.Sh SYNOPSIS
+.Nm
+.Op Fl plaintext
+.Op Fl overwrite
+.Op Fl k | kloset Ar location
```

Comment:
nit: *our* (read, openbsd) style is to document in the synopsis only the short letter if there is one.  (but afterwards in the description document both)